### ISSUE TITLE
Issue #8 PT-BR

### ISSUE THREAD

**Issue by VictorGarritano** [VictorGarritano/nlp_newsletter_translations#1]
### Research and Publications

- [x] Surveys
- [x] Visualizing NNs
- [x] Meta-Learning
- [x] NeRF
- [x] Introducing Dreamer

### Creativity, Ethics and Society

- [x] CORD-19
- [x] SECNLP
- [x] 3D Generative Design

### Tools and Datasets

- [x] Stanza
- [x] GridWorld
- [x] X-Stance
- [x] Textual Heatmaps

### Articles and Blog Posts

- [x] How to generate text
- [x] RoBERTa from scratch

### Education

- [x] Getting Started with JAX
- [x] NLP for Developers
- [x] Thomas Wolf

### Noteworthy Mentions

- [x] Google Sheets
- [x] New York NLP
- [x] PyTorch Lightning Tutorial
- [x] Batch Normalization
- [x] Long list of Books
